Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -0,0 +1,31 @@
+ BTWK_story_event.0001.a:0 "I must ensure my realm is prepared for my passing."
+
+ BTWK_story_event.0002.t:0 "Uther's Sickness"
+ BTWK_story_event.0002.desc:0 "Never being quite able to shake your ailments, your health is nearing its breaking point. It's a surprise you've managed to hold on this long, although you'd rather you hadn't. The pneumonia has left you completely incapacitated and bedridden. Unable to even perform the most basic of tasks, you have elected (getdiarch) to serve as regent in your dying days."
+ BTWK_story_event.0002.a:0 "I have managed all I can. May the Gods be merciful."
+
+ BTWK_story_event.0005.t:0 "Death of King Uther Pendragon"
+ BTWK_story_event.0005.desc:0 "Uther Pendragon, the King of Dumnonia, is dead. The only legitimate heir to the throne is his clubfooted grandson Mordred, a mere infant. Upon receiving news of his father's death, Uther's bastard son Arthur has returned from the Kingdom of Benoic and has been appointed Lord of Durnac by the current regent."
+ BTWK_story_event.0005.a:0 "Hail to King Mordred and Lord Arthur!"
+ BTWK_story_event.0005.b:0 "Let us hope Arthur drives the Saxons back to whence they came!"
+ BTWK_story_event.0005.c:0 "They are weak. We will conquer their lands, no matter their leader."
+ BTWK_story_event.0005.d:0 "This is interesting news."
+
+ BTWK_story_event.0006.t:0 "From One Pendragon to Another"
+ BTWK_story_event.0006.desc:0 "Your grandfather, King Uther Pendragon, has died. Prince Arthur, his bastard son and your uncle, has returned to Dumnonia from the Kingdom of Benoic upon hearing of Uther's death, where he had been sent to assist King Ban in his wars against the Franks. However, upon his return, your current regent stripped from you the lands of Durnac, rightfully yours after Uther's passing, appointing Arthur as their Lord. He has been hailed, like his father, as #italic Pendragon #. "
+ BTWK_story_event.0006.a:0 "A loyal vassal... or is he plotting for my throne?"
+ BTWK_story_event.0006.b:0 "He may truly be the one destined to save #italic Britannia - The Winter King. #"
+
+ BTWK_story_event.0007.t:0 "Arthur Pendragon"
+ BTWK_story_event.0007.desc:0 "Arthur Pendragon is a man of many virtues and many flaws. Banished from Dumnonia after the death of Uther's only legitimate son, Mordred, in battle, Arthur served in the court of his brother-in-law, King Budic of Broceliande. With his father's passing, however, Arthur was obliged to return from Armorica to ensure the stablilty of the realm for his infant nephew, Mordred ap Mordred. Will you fulfil your oath to Uther as Mordred's guiding hand, or instead take #italic other steps# to secure Dumnonia against the inevitable Saxon onslaught?"
```

Comment:
Implies that Arthur returned from Broceliande, but I was under the impression that he was stationed in Benoic when he left?

Reply:
I thought it felt off, I'd copied it from the bookmark description of Arthur seeing as it wasn't being used anymore